src/session/components/chat_member_row.rs

```diff
@@ -0,0 +1,86 @@
+use crate::tdlib::ChatMember;
+use gettextrs::gettext;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{glib, CompositeTemplate};
+
+use super::Avatar;
+use crate::{expressions, strings};
+use tdlib::enums::{UserStatus, UserType};
+
+mod imp {
+    use super::*;
+    #[derive(Debug, Default, CompositeTemplate)]
+    #[template(resource = "/com/github/melix99/telegrand/ui/components-chat-member-row.ui")]
+    pub(crate) struct ChatMemberRow {
+        #[template_child]
+        pub(super) avatar: TemplateChild<Avatar>,
+        #[template_child]
+        pub(super) user_name_label: TemplateChild<gtk::Label>,
+        #[template_child]
+        pub(super) member_status_label: TemplateChild<gtk::Label>,
+        #[template_child]
+        pub(super) user_status_label: TemplateChild<gtk::Inscription>,
+    }
+
+    #[glib::object_subclass]
+    impl ObjectSubclass for ChatMemberRow {
+        const NAME: &'static str = "ComponentsChatMemberRow";
```

Comment:
I think this widget is only useful for the `ChatInfoWindow`, so I don't think it's useful to put it inside the contents directory (usually these are shared widgets). I suggest turning the current `chat-info-window` module in a module directory and putting this widget inside that directory.